docs/labs/index.rst

```diff
@@ -18,9 +18,9 @@ If we get lots of positive feedback about any one of these projects then we will
 Flocker Docker plugin with CLI and GUI
 ======================================
 
-By way of example, here is a 55 second demo of the :ref:`Flocker Docker plugin <labs-docker-plugin>` provisioning portable Flocker volumes and moving them between hosts directly from the Docker CLI.
+By way of example, here is a 55 second demo of the :ref:`Flocker Docker plugin <docker-plugin>` provisioning portable Flocker volumes and moving them between hosts directly from the Docker CLI.
```

Comment:
I considered updating all these references, and didn't. Here's why:

* The labs plugin is actually a different project, different code, used differently. It follows that it has a different name.
-- or --
* Now that FDP is release, all these docs should be blown away / ported to main docs.

On balance I prefer the latter.